src/models/User.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const { sequelize } = require('../utils/db');
+const { DataTypes } = require('sequelize');
+
+const User = sequelize.define('User', {
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+}, {
+  tableName: 'users',
+  updatedAt: false,
```

Comment:
You should include the `timestamps` property and set it to false to disable the `updatedAt` and `createdAt` fields. 🌟